Lib/test/test_ctypes/test_struct_fields.py

```diff
@@ -55,11 +55,21 @@ class X(Structure):
         x.char = b'a\0b\0'
         self.assertEqual(bytes(x), b'a\x00###')
 
-    def test_6(self):
+    def test_cfield_instantiation(self):
         class X(Structure):
             _fields_ = [("x", c_int)]
         CField = type(X.x)
-        self.assertRaises(TypeError, CField)
+        self.assertRaisesRegex(TypeError,
+                               "cannot create '_ctypes.CField' instances",
+                               CField)
+
+    def test_cfield_immutability(self):
```

Comment:
`CField` is a basic object (inherif from `object`, no `metaclass`), so not add the inherif test.

Reply:
Well, you can test that :)